### PR TITLE
Changed holdtype on pistols to better match animations

### PR DESCRIPTION
**Pull request by SvveetMavis**
changed the holdtype from "pistol" to "revolver"  to better match the 2 hands in the viewmodel

**Before**
image

**After**
image2

### CODE REVIEW

**Review by saibotk — APPROVED** [review]
Nice could you add a small line to the Changelog.md? Otherwise good to go thank you very much 🙏

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - switched from `player.GetAll()` to `select(2, player.Iterator())`
   - use `net.ReadPlayer` / `net.WritePlayer` if applicable instead of `net.Read|WriteEntity`
   - Reduced radar bit size for net message
-
+- The holdtype for pistol weapons now matches the viewmodel
```
Comment (Histalek) [inline CHANGELOG.md:35]:
```suggestion
  - The holdtype for pistol weapons now matches the viewmodel

```

**Review by Histalek — APPROVED** [review]
Nice!

**Comment by TimGoll** [comment]
We should probably update this for the sidekick and the deputy as well so one can't spot the difference

**Comment by SvveetMavis** [comment]
I went through all the roles I could find with a deagle and sent in a PR for to change holdtype, so hopefully those get merged

**Pull request merged**